src/cli/serve/serve.js

```diff
@@ -250,9 +250,9 @@ export default function (program) {
         const settings = transformDeprecations(getCurrentSettings());
         const config = new Config(kbnServer.config.getSchema(), settings);
 
-        kbnServer.server.log(['info', 'config'], 'Reloading logging configuration due to SIGHUP.');
-        await kbnServer.applyLoggingConfiguration(config);
-        kbnServer.server.log(['info', 'config'], 'Reloaded logging configuration due to SIGHUP.');
+        // kbnServer.server.log(['info', 'config'], 'Reloading logging configuration due to SIGHUP.');
```

Comment:
Oh yeah, this is all coming back to me. We need to get off our fork at https://github.com/bevacqua/even-better/commit/4342ed5549e70ac46b34f93ade2b5d067cef30d5

Reply:
Yep, I got some good details from Court on this. Going to go with getting everything working and then re-implement the things that depend on good with either a new version or something else.